packages/nexys/src/classes/core/index.ts

```diff
@@ -294,12 +294,15 @@ export class Core {
    *  // Set application version (likely to be your app version)
    *  // This config is MUST-to-do because we will analyze each of your versions
    *  config.setAppVersion("1.0.0");
+   *  // This config is optional. If you want to identify your logs where came from, you can set platform
```

Comment:
If you want to identify your logs which platforms they came from, you can set platform config. olarak değiştirelim yazıyı.